### PR TITLE
Use oca_dependencies.txt, actualize .travis.yml

### PR DESCRIPTION
**Pull request by guewen**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,19 +14,21 @@ python:
...
-  - $HOME/maintainer-quality-tools/travis/travis_install_nightly 7.0
-  - pip install coveralls flake8
-  - git clone https://github.com/OCA/stock-logistics-barcode -b 7.0 $HOME/stock-logistics-barcode
+env:
+  matrix:
+  - LINT_CHECK="1"
+  - TESTS="1" ODOO_REPO="odoo/odoo"
+  - TESTS="1" ODOO_REPO="OCA/OCB"
+  global:
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0"
```
Comment (pedrobaeza) [inline .travis.yml]: This should be 7.0

Comment (guewen) [inline .travis.yml]: done

**Comment by dreispt** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**